src/shared/components/palette/address/write-address-field.html

```diff
@@ -1,36 +1,39 @@
 <div class="form-group" [id]="id()">
 
-  <h3 class="heading-small heading-17">{{caseField | ccdFieldLabel }}</h3>
+  <div *ngIf="!isComplexWithHiddenFields()">
```

Comment:
for my knowledge, why we want to show the components below only if there are no hidden fields?

Reply:
This is the postcode lookup - 'Find Address' button and we don't want to see it if we selectively choose fields to show from the address using the new functionality.

Reply:
got it, cheers